acoustid/data/fingerprint.py

```
@@ -79,44 +80,42 @@ def _create_search_query(
+        query = (
+            select(*columns)
+            .join(schema.track, schema.track.c.id == f.c.track_id)
+            .where(f.c.score > self.min_score)
+            .order_by(f.c.score.desc(), f.c.id)
         )
+
         if max_results:
             query = query.limit(max_results)
         return query
 
     def _search_index(self, fp, length, index, max_candidates=None, min_score_pct=None):
```

Ruff (0.8.2)

Missing return type annotation for private function `_search_index`

(ANN202)